packages/protocol/contracts/common/MetaTransactionWallet.sol

```diff
@@ -260,6 +262,21 @@ contract MetaTransactionWallet is
     return (returnValues, returnLengths);
   }
 
+  /**
+   * @notice Transfers all cUSD to the `signer`.
+   * @dev Valora v1.2 ignores the `walletAddress` of the user, which may result in cUSD being
+   *   sent to users MetaTransactionWallets instead of their EOA. This function allows that cUSD
+   *   to be transferred to the EOA without requiring action by the user.
+   * @return Whether or not the cUSD transfer succeeded.
+   */
+  function transferCeloDollarsToSigner() external returns (bool) {
+    bytes32 STABLE_TOKEN_REGISTRY_ID = keccak256(abi.encodePacked("StableToken"));
+    IRegistry registry = IRegistry(0x000000000000000000000000000000000000ce10);
+    IERC20 stable = IERC20(registry.getAddressForOrDie(STABLE_TOKEN_REGISTRY_ID));
```

Comment:
Why not use UsingRegistry for this?

Reply:
I wanted to avoid adding a storage variable for something that might be deprecated in the near future.

Reply:
Makes sense. Make the StableToken Registry ID a constant rather than computing it on each call?

Reply:
Why not make this `transferERC20ToSigner(address)`? So we can pass in arbitrary ERC20 contracts and get celo support as well.

Reply:
Yeah great suggestion